tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1546,6 +1546,31 @@ std::ostream& operator<<(std::ostream& os, const TestParamBase& p) {
   return os;
 }
 
+template<typename T>const char *format();
+
+template<> const char *format<int>()      { return "%d"; }
+template<> const char *format<float>()    { return "%.1f"; }
+
+template<typename T>
+void out_vector(const std::vector<T> &vector, const char *pComment, const string &name, const string &type="") {
+
+  char buffer[256], *pBuff = buffer;
+  const auto len_buffer = sizeof(buffer);
+  pBuff += snprintf(pBuff, len_buffer, "%d, d=", vector.size());
+  for (int i = 0; i < vector.size(); i++) {
+    if (i)
+      pBuff += snprintf(pBuff, len_buffer-(pBuff-buffer), ",");
+    pBuff += snprintf(pBuff, len_buffer-(pBuff-buffer), format<T>(), vector[i]);
+  }
```

Comment:
Can you use absl::StrJoin and absl::StrCat to implement this functionality?

Reply:
Done, see latest commit